war/.prettierignore

```diff
@@ -34,3 +34,5 @@ src/main/webapp/jsbundles/
 
 # TODO not sure why this fails
 src/main/less/abstracts/theme.less
+
+../.github/PULL_REQUEST_TEMPLATE.md
```

Comment:
I went ahead and excluded the file from prettier, because we can't use the double slashes for single line exclusions, given it's written in markdown.

Reply:
I think it would be preferable to change the first argument to `.gsub()` in https://github.com/jenkinsci/core-changelog-generator/blob/c13ec5065e4c16b836edc15d7d0a2b3ddd22f6ac/generate-jenkins-changelog.rb#L145 from `"\*"` to `"-"` and retain consistency for all Markdown files in this repository (dashes everywhere without a Prettier exclusion) than to retain the existing code in `generate-jenkins-changelog.rb` at the cost of decreasing consistency within this repository by using stars in `PULL_REQUEST_TEMPLATE.md` but dashes everywhere else (and adding a Prettier exclusion).

Reply:
Yup, awesome pointer! I looked for that yesterday, but wasn't able to locate it for some reason.